Apps/Src/Task_Init.c

```diff
@@ -0,0 +1,184 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+#include "os.h"
+#include "config.h"
+#include "Tasks.h"
+#include "CAN_Queue.h"
+
+#ifndef SIMULATION
+#include "stm32f4xx.h"
+#endif
+
+void Task_Init(void *p_arg) {
+
+#ifndef SIMULATION
+	OS_CPU_SysTickInit(SystemCoreClock / (CPU_INT32U) OSCfg_TickRate_Hz);
+#endif
+
+    OS_ERR err;
+    OSSemCreate(&Fault_Sem4,
+                "Fault/Tripped Semaphore",
+                0,
+                &err);
+    assertOSError(err);
+
+    OSSemCreate(&SafetyCheck_Sem4,
+                "Safety Check Semaphore",
+                0,
+                &err);
+    assertOSError(err);
+
+    OSMutexCreate(&WDog_Mutex,
+                "Watchdog Mutex",
+                &err);
+    assertOSError(err);
+
+        //1
+    	OSTaskCreate(&FaultState_TCB,				// TCB
```

Comment:
We should be calling assertOSError after each of these creations.

Reply:
Updated in #428